test/JDBC/input/views/sys-syscolumns.sql

```diff
@@ -1,4 +1,4 @@
--- sla 16000
+-- sla 20000
```

Comment:
Where are we testing the privileges that display that schema permissions is really redundant?

Reply:
@tanyagupta17 I think the tests you have added are already presents in multiple files. Here, we want to validate that `has_schema_privilege(USAGE)`  checks will always return true for all TSQL schemas and TSQL roles. So, you can invoke this function from PSQL endpoint in tests to validate the same.

Reply:
Done. Added the test case to check that all tsql users have USAGE privilege on tsql schemas